packages/preview/cheda-seu-thesis/0.3.1/init-files/bachelor_thesis.typ

```diff
@@ -0,0 +1,310 @@
+#import "../seu-thesis/lib.typ": bachelor-conf, bachelor-utils

```

Comment:
Can you please restore the "absolute" import here (i.e `@preview/cheda-seu-thesis:0.3.1`)?